content/en/docs/reference/using-api/server-side-apply.md

```diff
@@ -224,16 +224,76 @@ merging, see
-| `//+listMapKey` | `x-kubernetes-list-map-keys` | Slice of map keys that uniquely identify entries for example `["port", "protocol"]` | Only applicable when `+listType=map`. A slice of strings whose values in combination must uniquely identify list entries. While there can be multiple keys, `listMapKey` is singular because keys need to be specified individually in the Go type. | 1.16 |
+| `//+listType` | `x-kubernetes-list-type` | `atomic`/`set`/`map` | Applicable to lists. `set` only applies to lists with scalar elements only. These elements must be unique. `map` applies to lists of nested types only. The key values (see `listMapKey`) must be unique in the list. `atomic` can apply to any list. If configured as `atomic`, the entire list is completely replaced during merge; a single manager manages the list as a whole at any one time. If `set` or `map`, different managers can manage entries separately. | 1.16          |
+| `//+listMapKey` | `x-kubernetes-list-map-keys` | List of field names, e.g. `["port", "protocol"]` | Only applicable when `+listType=map`. A list of field names whose values uniquely identify entries in the list. While there can be multiple keys, `listMapKey` is singular because keys need to be specified individually in the Go type. The key fields must be scalars. | 1.16 |
 | `//+mapType` | `x-kubernetes-map-type` | `atomic`/`granular` | Applicable to maps. `atomic` means that the map can only be entirely replaced by a single manager. `granular` means that the map supports separate managers updating individual fields. | 1.17 |
 | `//+structType` | `x-kubernetes-map-type` | `atomic`/`granular` | Applicable to structs; otherwise same usage and OpenAPI annotation as `//+mapType`.| 1.17 |
 
+If `listType` is missing, a `patchMergeStrategy=merge` marker will be
+converted into a `listType=map` and the corresponding `patchMergeKey`
+marker will be converted into `listMapKey`.
+
+`atomic` is always recursive and these topology markers won't be
+evaluated within an atomic type.
+
+These markers are specified as comments and don't have to be repeated as
+field tags.
+
+#### Compatibility across topology changes
+
+Changing the topology of types that have existing objects will have
+different consequences depending on the type of change.
+
+When the `listType`, `mapType`, or `structType` changes from
+`map`/`set`/`granular` to `atomic`, the whole list, map or struct of
+existing objects will end-up being owned by actors who owned an element
+of these types. This means that any further change to these objects
+would cause a conflict.
+
+When a list, map, or struct goes from `atomic` to `map`/`set`/`granular`,
```

Comment:
I don't know what "with that structure" refers to?
Why is there no conflict?

When a list, map, or struct changes from `atomic` to `map`/`set`/`granular`, the object does not conflict when new changes ... ?

Reply:
Lines 261-264 could use some clarification.
When a list, map, or struct changes from `atomic` to `map`/`set`/`granular`, the API server saves this state change(?).
Any future change to objects with that structure [what structure] do not produce conflicts.